src/commands/apply/index.ts

```diff
@@ -98,5 +128,9 @@ export default class Apply extends Command {
   protected async finally(): Promise<any> {
     await this.writeFileLog()
     this.log(renderFilePaths({ logFilePath: this.logFilePath }))
+
+    // analyticsCloseAndFlush has a very short timeout because it will
+    // otherwise trigger a rerender of the listr tasks on error exits
+    await Promise.allSettled([Sentry.close(2000), analyticsCloseAndFlush(2000)])
```

Comment:
The comment is outdated and should be removed (both here and in create/index). This relates to a workaround where we put `1` as param to `analyticsCloseAndFlush`.